docs/Modules/BatchedSession.mdx

```diff
@@ -58,4 +58,4 @@ const sessionRouterModule = await createBatchedSessionRouterModule({
 });
```

Comment:
in above is the routerModuleAddr same as DEFAULT_BATCHED_SESSION_ROUTER_MODULE and all of these are importable from  "@biconomy/account". can we change to DEFAULT_BATCHED_SESSION_ROUTER_MODULE and add import as well